packages/quantic/force-app/main/default/lwc/quanticQueryError/quanticQueryError.js

```diff
@@ -25,6 +25,8 @@ import { errorMap, genericError } from './errorLabels.js';
  * <c-quantic-query-error engine-id={engineId}></c-quantic-query-error>
  */
 export default class QuanticQueryError extends LightningElement {
+  static delegatesFocus = true;
```

Comment:
What is that supposed to do?

Reply:
This is related to the point you highlighted in a previous PR: https://github.com/coveo/ui-kit/pull/2558#discussion_r1038511134
Basically, the fallback copy to clipboard method will reset the focus to the beginning of the page. That's why I'm adding this logic in order to reset the focus to the copyToClipboard button using the [delegatesFocus](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.create_components_focus).